Code/Framework/AzCore/AzCore/DOM/DomPrefixTree.h

```diff
@@ -129,11 +129,18 @@ namespace AZ::Dom
         //! @return The DomPrefixTree that is detached.
         DomPrefixTree<T> DetachSubTree(const Path& path);
 
-        //! Attaches a subtree provided at the node that matches the provided path. Attaching will overwrite the node at the path.
+        //! Attaches a subtree at the provided path. Attaching will overwrite the subtree at the path.
```

Comment:
nit: Probably good to also add a comment that says it'll return false if the path couldn't be reached in the tree from the root.